internal/backends/winit/glwindow.rs

```diff
@@ -34,17 +34,33 @@ fn position_to_winit(pos: &corelib::api::WindowPosition) -> winit::dpi::Position
     }
 }
 
-fn size_to_winit(pos: &corelib::api::WindowSize) -> winit::dpi::Size {
```

Comment:
I'm not a fan of using trait for this kind of stuff where you just add one function to a type. I kind of prefer free floating function as before.  I don't really know why though.  But if you prefer trait, i'm ok with it.

Reply:
Good question, in reverse I don't know why I prefer a trait when ... your suggestion is literally less lines of code. I'll squash some of the other changes and change this one to use a free standing function. Thanks for the nudge :)